docs/bias_correction.rst

```diff
@@ -1,22 +1,84 @@
-Bias Correction **(IN PROGRESS)**
-=================================
+Bias Correction
+===============
 
 This page documents the implementation of
 **bmorph** bias correction for streamflow
 data in a watershed. An example workflow notebook
-can be found in *example_notebooks*.
+can be found in ``bmorph_tutorial``.
+
+bmorph Overview
```

Comment:
I think you need to back up a bit here, it's a bit too in the weeds right away - let's discuss what we need to say about bias correction offline.